app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -2,4 +2,8 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job < ManageIQ::P
   include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Job
 
   require_nested :Status
+
+  def retire_now(requester = nil)
+    finish_retirement
```

Comment:
We should still be setting the `retirement_requester` as part of the `retire_now` logic: https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/retirement_mixin.rb#L121